src/simplay-web/src/SimulationSpooler.ts

```diff
@@ -20,18 +20,18 @@ export interface Area {
 
 export interface EventUnserialized {
   action:
-    | "log"
-    | "system"
-    | "setDecoratingText"
-    | "setPosition"
-    | "setVisible"
-    | "setRemoveInteracting"
-    | "setCreateInteracting"
-    | "setNotInteracting"
-    | "moveNear"
-    | "moveNearPoint"
-    | "container.setLevel"
-    | "resource.setUtilization"; // TODO what else should be available?
+    | 'log'
+    | 'system'
+    | 'setDecoratingText'
+    | 'setPosition'
+    | 'setVisible'
+    | 'setRemoveInteracting'
+    | 'setCreateInteracting'
+    | 'setNotInteracting'
+    | 'moveNear'
+    | 'moveNearPoint'
+    | 'container.setLevel'
+    | 'resource.setUtilization'; // TODO what else should be available?
```

Comment:
These actions do not reflect what our architecture is, please create a task to fix it, or fix it yourself.